### ISSUE TITLE
Swipe gesture Initially not working in savedFilesFragment

### ISSUE THREAD

**Issue by riteshknayak** [GauthamAsir/WhatsApp_Status_Saver#22]
When the app is the first opened. If recyclerview in savedFilesFragment is Empty than swipe gesture doesn't work.

**Comment by riteshknayak** [comment]
I tried recreating the same fragment and transfered code little bit at a time. I found that `recyclerView.setLayoutManager(new GridLayoutManager(getActivity(), Common.GRID_COUNT));` adding this line causes the Issue. Can anyone explain how this causing the issue.

**Comment by GauthamAsir** [comment]
> When the app is the first opened. If recyclerview in savedFilesFragment is Empty than swipe gesture doesn't work.

Can you explain me the steps to re create, according to your previous message im unable to re create.

**Comment by riteshknayak** [comment]
By recreating, I mean, I created a fragment named `TestFragment` than changed portion of `MainActivity` and `PageAdapter` to include this Fragment as a new tab. 

At first I added the `SwipeRefreshLayout` as  expected it was working. Than I commented `SavedFilesFragment.java` and `fragment_saved_files.xml` . I didn't recreated the RecyclerView Adapter. Now just as I added these two lines `        recyclerView.setHasFixedSize(true);
        recyclerView.setLayoutManager(new GridLayoutManager(getActivity(), Common.GRID_COUNT));
`
The swipe gesture didn't worked

**Comment by GauthamAsir** [comment]
Current app version has issue or the app you trying to create has issue? Because At my end it works properly

**Comment by riteshknayak** [comment]
> > When the app is the first opened. If recyclerview in savedFilesFragment is Empty than swipe gesture doesn't work.
> 
> Can you explain me the steps to re create, according to your previous message im unable to re create.

[Link of screen recording](https://drive.google.com/file/d/1sqGyga4_oBglQ9KoA3PirioTDLeKALOh/view?usp=sharing)

**Comment by riteshknayak** [comment]
Current one has this issue

> Current app version has issue or the app you trying to create has issue? Because At my end it works properly

**Comment by GauthamAsir** [comment]
Okay can you do pull request if you found the solution will merge it

**Comment by riteshknayak** [comment]
Haven't found the solution yet.

**Comment by GauthamAsir** [comment]
Okay will look into it

**Comment by riteshknayak** [comment]
Did you got the Issue in your app

> Current app version has issue or the app you trying to create has issue? Because At my end it works properly



> > > When the app is the first opened. If recyclerview in savedFilesFragment is Empty than swipe gesture doesn't work.
> > 
> > 
> > Can you explain me the steps to re create, according to your previous message im unable to re create.
> 
> [Link of screen recording](https://drive.google.com/file/d/1sqGyga4_oBglQ9KoA3PirioTDLeKALOh/view?usp=sharing)

**Comment by GauthamAsir** [comment]
Yes, facing the issue but it isn't any major issue, so i didn't work on it. Will check it anyways

**Comment by riteshknayak** [comment]
Sorry Closed by mistake
